### ISSUE TITLE
Missing Items from Class Data

### ISSUE THREAD

**Issue by Road-block** [acelan/AceBIS#2]
A couple items that are present on the spreadsheet but not in the generated data

Pepe's Shroud of Pacification : 34010
Praetorian's Legguards : 32263

Are not registered on any BiS List for [protection warrior](https://github.com/acelan/AceBIS/blob/master/AceBIS/Data/ProtectionWarrior.lua)

There's likely to be more missing, I didn't cross reference everything on the spreadsheet against the data files, I just happened to have those and noticed the tooltip hint missing.

**Comment by Road-block** [comment]
Shaman are missing all data for ranged slot (totems) at least Elemental that I checked.

Edit: This is also true for the spreadsheet so I guess it's not an addon bug.

**Comment by acelan** [comment]
Yes, for the protection spec., it's hard to select correct gears.
Currently, I select the gears with defense rating only. There are many gears got better score than those with defense rating, I probably could add the gears with dodge rating and parry rating.

I'll check shaman's gears, thanks for reporting this.

**Comment by acelan** [comment]
For the relic(totem) for shaman, here is the explanation from the database maintainer
```
Fazers: 
because relics deal with specific abilities there wasnt a way to give it a simple ep value, thats why its not on any of the final sheets
```

And the gears for protection spec.(warrior/paladin) should be fixed in v3.0.5
https://github.com/acelan/AceBIS/commit/947a430a55c181cbdf60919e55730e34f99e144c
